django_hatstall/django_hatstall/settings.py

```diff
@@ -118,3 +118,4 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+STATICFILES_DIRS = ('django_hatstall/templates/static/',)
```

Comment:
I am not sure if this is the best place for static files. But we can live with and change it later if we find issues with this approach.